jdbi/src/main/java/org/killbill/commons/jdbi/argument/LocalDateArgumentFactory.java

```diff
@@ -48,10 +48,9 @@ public LocalDateArgument(final LocalDate value) {
         @Override
         public void apply(final int position, final PreparedStatement statement, final StatementContext ctx) throws SQLException {
             if (value != null) {
-                // ISO8601 format
-                statement.setString(position, value.toString());
+                statement.setDate(position, new java.sql.Date(value.toDate().getTime()));
```

Comment:
are there any potential side effects of this change for other engines such as Mysql?